tools/rpkg/R/backend-dbplyr__duckdb_connection.R

```diff
@@ -14,9 +14,6 @@
 #' x <- tibble(txt = c("why", "video", "cross", "extra", "deal", "authority"))
 #' dbx <- copy_to(con, x, overwrite = TRUE)
 #'
-#' # x %>% filter(grepl("^.[hrx]", txt))
-#' dbx %>% filter(grepl("^.[hrx]", txt))
```

Comment:
Do we have an idea what causes the re2 valgrind warnings?

Reply:
Yes but lets discuss in person